include/itkAttenuationImageFilter.hxx

```diff
@@ -0,0 +1,225 @@
+/*=========================================================================
+ *
+ *  Copyright NumFOCUS
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+#ifndef itkAttenuationImageFilter_hxx
+#define itkAttenuationImageFilter_hxx
+
+#include "itk_eigen.h"
```

Comment:
Is this enough? Don't we also need `#include ITK_EIGEN(Dense)`?

Reply:
It compiles for me, but I'm having a hard time pinning down where "Eigen/Dense" is actually being included, so I agree that it should be included here.

Is there anything else that should be added to `itk-module.cmake` as well to indicate that we have a dependency on Eigen?

Reply:
`#include ITK_EIGEN(Dense)` is included in fixup 2.

Reply:
You are right: we should probably add `ITKEigen3` to `itk-module.cmake`.